src/chatlist.rs

```diff
@@ -130,6 +120,14 @@ impl Chatlist {
                 .map_err(Into::into)
         };
 
+        let skip_id = if flag_for_forwarding {
```

Comment:
The code seems to be accidentally moved during rebasing, but looks ok otherwise.